### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Auth MFA docs
https://dbtlabs.atlassian.net/browse/ENTERPRISE-1323

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

File: website/docs/docs/cloud/manage-access/mfa.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+---
+
+dbt Cloud provides multiple options for multi-factor authentication (MFA). MFA provides an additional layer of security on top of username and password. The available MFA methods are:
+- SMS verification code
+- Authenticator app
+- Webauthn-compliant security key
+
+## Configuration
+
+You may have one of the three MFA methods configured per user. These are enabled at the user level, not the account level.
```
Comment (matthewshaver) [inline website/docs/docs/cloud/manage-access/mfa.md]:
```suggestion
You can only have one of the three MFA methods configured per user. These are enabled at the user level, not the account level.
```

File: website/docs/docs/cloud/manage-access/mfa.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+## Configuration
+
+You can only have one of the three MFA methods configured per user. These are enabled at the user level, not the account level.
+
+1. Navigate to the **Account settings** and under **Your profile** click on **Password & Security**. Click **Enroll** next to the preferred method.
+
+<Lightbox src="/img/docs/dbt-cloud/mfa-enrollment.png" title="Select MFA enrollment method" />
+
+Choose the next steps based on your enrollment selection:
```
Comment (matthewshaver) [inline website/docs/docs/cloud/manage-access/mfa.md]:
```suggestion
Choose the next steps based on your preferred enrollment selection:
```

File: website/docs/docs/cloud/manage-access/mfa.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+3. You will receive an SMS message with a six digit code. Enter the code in dbt Cloud.
+
+<Lightbox src="/img/docs/dbt-cloud/enter-code.png" title="Enter the 6-digit code" />
+
+</expandable>
+
+<expandable alt_header="Authenticator app" >
+
+2. Open your preffered authentication app (like Google Authenticator) and scan the QR code.
```
Comment (ajaythomas) [inline website/docs/docs/cloud/manage-access/mfa.md]: nit: preferred

**Review by ajaythomas — APPROVED** [review]

File: website/docs/docs/cloud/manage-access/mfa.md (hunk 3 above)
Comment (matthewshaver) [reply website/docs/docs/cloud/manage-access/mfa.md]: Good catch!
```suggestion
2. Open your preferred authentication app (like Google Authenticator) and scan the QR code.
```

**Pull request merged**